docs/guide/http_cache.md

```diff
@@ -241,14 +248,22 @@ fastcgi_param HTTPCACHE_PURGE_SERVER "http://varnish:80";
 
 # Configure IP of your Varnish server to be trusted proxy
 # Replace fake IP address below by your Varnish IP address
-fastcgi_param SYMFONY_TRUSTED_PROXIES "193.22.44.22";
+fastcgi_param TRUSTED_PROXIES "193.22.44.22";
 ```
 
 !!! caution "Trusted proxies when using SSL offloader / loadbalancer in combination with Varnish"
 
-    If your installation works behind Varnish and SSL offloader (like HAProxy), you need to add `127.0.0.1` to `SYMFONY_TRUSTED_PROXIES`.
+    If your installation works behind Varnish and SSL offloader (like HAProxy), you need to set the `TRUSTED_PROXIES` env var:
+    
+    ```
+    # .env
+    TRUSTED_PROXIES=127.0.0.1,REMOTE_ADDR
```

Comment:
I am not sure if REMOTE_ADDR must and should be used here, it was not required before and afaik it is still only required for installation on cloud (platform.sh) (when you can not define ip address for varnish auth).